scripts/install/install_bmi.sh

```diff
@@ -0,0 +1,64 @@
+set -x
+
+### START BMI setup
+
+TOP_DIR=$(cd $(dirname "$0") && pwd)
+
+# Move to main ims directory for install from script location
+pushd $TOP_DIR/../../
+
+### Install BMI
+
+user=`whoami`
+sudo mkdir -p /var/log/bmi/logs /etc/bmi /opt/bmi
+sudo chown -R $user:$user /etc/bmi /var/log/bmi /opt/bmi
+sudo chmod 775 /etc/bmi /var/log/bmi /opt/bmi
```

Comment:
We don't need to chmod here, just chowning is sufficient.

Also /var/lib/tftpboot and /var/lib/tftpboot/pxelinux.cfg also need to be chowned.